<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(36, 31, 49);
	color: white;                        /* Цвет текста */
}

QTextEdit {
    background-color: rgb(35, 35, 45);  /* Цвет фона для QTextEdit */
    color: white;                        /* Цвет текста в QTextEdit */ */
}

QPushButton {
    background-color: rgb(45, 40, 60);  /* Цвет фона для кнопок */
    color: white;                        /* Цвет текста для кнопок */
  
}

QListWidget {
    background-color: rgb(30, 30, 40);  /* Цвет фона для списка */
    color: white;                        /* Цвет текста для списка */

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addNote">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../images/icon.qrc">
              <normaloff>:/icon/addNote.png</normaloff>:/icon/addNote.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addFolder">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../images/icon.qrc">
              <normaloff>:/icon/addFolder.png</normaloff>:/icon/addFolder.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="styleSheet">
           <string notr="true">QWidget {

background-color: rgb(23, 21, 30); /* Цвет фона */

color: white; /* Цвет текста */

border: 2px solid rgb(45, 40, 60); /* Цвет рамки */

padding: 10px; /* Отступы */

}

QTextEdit {
    background-color: rgb(35, 35, 45);  /* Цвет фона для QTextEdit */
    color: white;                        /* Цвет текста в QTextEdit */
    border: 1px solid rgb(45, 40, 60);   /* Цвет рамки для QTextEdit */
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {

background-color: rgb(27, 25, 38); /* Цвет фона */

color: white; /* Цвет текста */

}</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="rightListWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {

background-color: rgb(23, 21, 30); /* Цвет фона */

color: white; /* Цвет текста */

border: 2px solid rgb(45, 40, 60); /* Цвет рамки */

padding: 10px; /* Отступы */

}</string>
       </property>
       <property name="currentRow">
        <number>-1</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="title">
     <string>Bookmarks</string>
    </property>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuSearch"/>
   <addaction name="menuBookmarks"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../images/icon.qrc"/>
 </resources>
 <connections/>
</ui>
